content/zh/docs/concepts/architecture/garbage-collection.md

```diff
@@ -0,0 +1,363 @@
+`EndpointSlice` objects. The Service uses *labels* to allow the control plane to
+determine which `EndpointSlice` objects are used for that Service. In addition
+to the labels, each `EndpointSlice` that is managed on behalf of a Service has
+an owner reference. Owner references help different parts of Kubernetes avoid
+interfering with objects they don’t control.
+-->
+属主关系与某些资源所使用的的[标签和选择算符](/zh/docs/concepts/overview/working-with-objects/labels/)
+不同。例如，考虑一个创建 `EndpointSlice` 对象的 {{<glossary_tooltip text="Service" term_id="service">}}
+对象。Service 对象使用*标签*来允许控制面确定哪些 `EndpointSlice` 对象被该
+Service 使用。除了标签，每个被 Service 托管的 `EndpointSlice` 对象还有一个属主引用属性。
+属主引用可以帮助 Kubernetes 中的不同组件避免干预并非由它们控制的对象。
+
+{{< note >}}
+<!--
+Cross-namespace owner references are disallowed by design.
+Namespaced dependents can specify cluster-scoped or namespaced owners.
+A namespaced owner **must** exist in the same namespace as the dependent.
+If it does not, the owner reference is treated as absent, and the dependent
+is subject to deletion once all owners are verified absent.
+-->
+根据设计，系统不允许出现跨名字空间的属主引用。名字空间作用域的依赖对象可以指定集群作用域或者名字空间作用域的属主。
+名字空间作用域的属主**必须**存在于依赖对象所在的同一名字空间。
+如果属主位于不同名字空间，则属主引用被视为不存在，而当检查发现所有属主都已不存在时，
+依赖对象会被删除。
+
+<!--
+Cluster-scoped dependents can only specify cluster-scoped owners.
+In v1.20+, if a cluster-scoped dependent specifies a namespaced kind as an owner,
+it is treated as having an unresolvable owner reference, and is not able to be garbage collected.
+-->
+集群作用域的依赖对象只能指定集群作用域的属主。
+在 1.20 及更高版本中，如果一个集群作用域的依赖对象指定了某个名字空间作用域的类别作为其属主，
+则该对象被视为拥有一个无法解析的属主引用，因而无法被垃圾收集处理。
+
+<!--
+In v1.20+, if the garbage collector detects an invalid cross-namespace `ownerReference`,
+or a cluster-scoped dependent with an `ownerReference` referencing a namespaced kind, a warning Event 
+with a reason of `OwnerRefInvalidNamespace` and an `involvedObject` of the invalid dependent is reported.
+You can check for that kind of Event by running
+`kubectl get events -A --field-selector=reason=OwnerRefInvalidNamespace`.
+-->
+在 1.20 及更高版本中，如果垃圾收集器检测到非法的跨名字空间 `ownerReference`，
+或者某集群作用域的依赖对象的 `ownerReference` 引用某名字空间作用域的类别，
+系统会生成一个警告事件，其原因为 `OwnerRefInvalidNamespace`，`involvedObject`
+设置为非法的依赖对象。你可以通过运行
+`kubectl get events -A --field-selector=reason=OwnerRefInvalidNamespace`
+来检查是否存在这类事件。
+{{< /note >}}
+
+<!--
+## Cascading deletion {#cascading-deletion}
+
+Kubernetes checks for and deletes objects that no longer have owner
+references, like the pods left behind when you delete a ReplicaSet. When you
+delete an object, you can control whether Kubernetes deletes the object's
+dependents automatically, in a process called *cascading deletion*. There are
+two types of cascading deletion, as follows: 
+
+* Foreground cascading deletion
+* Background cascading deletion
+-->
+## 级联删除    {#cascading-deletion}
+
+Kubernetes 会检查并删除那些不再拥有属主引用的对象，例如在你删除了 ReplicaSet
+之后留下来的 Pod。当你删除某个对象时，你可以控制 Kubernetes 是否要通过一个称作
+*级联删除（Cascading Deletion）*的过程自动删除该对象的依赖对象。
```

Comment:
italic format not worked，need an extra space both start and end position?

Reply:
I'm gonna drop this italic format which renders ugly in Chinese.